app/templates/src/static/css/cf-theme-overrides.less

```diff
@@ -134,9 +134,17 @@
 // .success
 @input-success:                 @green;
 
+// .disabled
+@input-disabled:                 @gray-10;
```

Comment:
Should probably keep this one or [this one](https://github.com/cfpb/generator-cf/pull/102/files#diff-135a296121653a13551c1aeb70adc9b0R138), although they may just cancel each other out once one of these is merged.